include/usb/usb_device.h

```diff
@@ -112,6 +112,9 @@ typedef void (*usb_ep_callback)(u8_t ep,
  * For device to host class requests, 'len' and 'payload_data' should be
  * set by the callback function with the length and the address of the
  * data to be transmitted buffer respectively.
+ *
+ * @return 0 on success, positive value if request is intended to be handled
```

Comment:
@return 0 on success, `-EINVAL` if the request has not been handled by the custom handler and instead needs to be handled by the core USB stack. Any other error code to denote failure within the custom handler.

Reply:
I took this from callback doc and put only for `custom_handler` when we need that. No sense to change every type of handlers there.